### DISCUSSION TITLE
Opening Geany in a preset windows size

### DISCUSSION THREAD

**Discussion by ineuw** [geany/geany#3940] [General]
I need to open Geany in a predefined window size. This would solve my issue of line wrap at a particular column width. The OS is Linux Mint Cinnamon 21/22. 

![geany settings for line wrap](https://github.com/user-attachments/assets/baed76e4-f74b-4857-a48e-4d6e0be50174)

**Comment by elextr** [comment]
So long as you have `Edit->Preferences->General->Startup->Shutdown->Save Window size` Geany should restart at the size you dragged it to before shutdown.  Note that this depends on the window manager and GTK providing the correct size.  It is a known issue that some desktops provide a size that is slightly smaller, so the size changes over time.  AFAIK Cinnamon works right.

**Comment by ineuw** [comment]
Thanks. It retains the window from session to session in LM Cinnamon 22. It was inconsistent previously.
